### PR TITLE
Desperate prayer

### PR DESCRIPTION
**Pull request by HawkCorrigan**
Some check it against edge please.

### CODE REVIEW

File: src/parser/priest/shared/CHANGELOG.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import {
+  HawkCorrigan,
+} from 'CONTRIBUTORS';
+import { change, date } from 'common/changelog';
+
+export default [
+  change(date(2019, 12, 19), <>Fixed some ordering issues in the DP module.</>, [HawkCorrigan]),
```
Comment (MartijnHols) [inline src/parser/priest/shared/CHANGELOG.js]: This changelog is unused. Please add it to the changelog of all Priest specs (so 3 entries).

In addition please use <SpellLink /> for DP as not everyone may understand it.

Comment (abelito75) [reply src/parser/priest/shared/CHANGELOG.js]: I'm a bit late but shadow doesn't have access to Desperate prayer so plz no change log there

**Comment by abelito75** [comment]
Is this a dead pr?

**Comment by HawkCorrigan** [comment]
ye propably
I cant test it properly and it doesnt look like anyone cares too much about edge anyway

**Pull request closed without merging**